triton/tut/interactive.rst

```diff
@@ -247,8 +247,8 @@ Exercises
 
    c) Tell srun to use five CPUs (``-c 5``).  Does it go any faster?
 
-   d) Use the ``--threads=5`` option to the Python program to tell it
-      to also use five threads.  ``... python .../pi.py --threads=5``
+   d) Use the ``--nprocs=5`` option to the Python program to tell it
+      to also use five processes.  ``... python .../pi.py --nprocs=5``
```

Comment:
C and D have been removed in the latest - to be moved to an parallel exercise.  I'm adding a note here so we won't lose this, and we can make sure it's there later.  Removing in the conflict resolution.